src/EditorFeatures/TestUtilities/Diagnostics/DiagnosticProviderTestUtilities.cs

```diff
@@ -16,28 +16,31 @@ public static class DiagnosticProviderTestUtilities
             Workspace workspace,
             Document document,
             TextSpan span,
-            bool includeSuppressedDiagnostics = false)
+            bool includeSuppressedDiagnostics = false,
+            bool includeNonLocalDocumentDiagnostics = false)
```

Comment:
so... with optional parameters like thsi, i basically have no idea what the end result is in terms of tests wrt what value we actually have here.  Is there a reason we're not just flipping this on for all tests and just always doing htis?

If some things need an exception, i'd rathe they be explicit about passing in false.

Reply:
📝 Also note that no new tests should be using these classes.